.golangci.yaml

```diff
@@ -64,4 +64,25 @@ linters:
     - scopelint         # deprecated since v1.39.0 - replaced by 'exportloopref'.
     - rowserrcheck      # disabled because of generics - https://github.com/golangci/golangci-lint/issues/2649.
     - sqlclosecheck     # disabled because of generics - https://github.com/golangci/golangci-lint/issues/2649.
-    - wastedassign      # disabled because of generics - https://github.com/golangci/golangci-lint/issues/2649.
\ No newline at end of file
+    - wastedassign      # disabled because of generics - https://github.com/golangci/golangci-lint/issues/2649.
+
+linters-settings:
+  goheader:
+    template: |
+      Copyright 2018-2022 CERN
```

Comment:
Instead of 2022 you can use the built-in value `{{year}}` (https://github.com/denis-tingaikin/go-header#bult-in-values)